.github/workflows/ci.yml

```diff
@@ -57,3 +58,19 @@ jobs:
 
     - name: Docs
       run: cargo doc
+
+  no_std_build:
+    name: "Ensure no_std can build"
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
+      with:
+        fetch-depth: 0
+    - uses: actions-rs/toolchain@v1
+      with:
+        toolchain: stable
+        profile: minimal
+        override: true
+    
+    - name: Build
+      run: cargo build -p libsignify
```

Comment:
The best way to ensure a no_std library builds without libstd is to use a target that doesn't have libstd ([my trusted source: an embedded dev](https://twitter.com/bitshiftmask/status/1472614170663305226)).

So the easiest is probably:

```
rustup target add thumbv6m-none-eabi
cargo build -p libsignify --target thumbv7em-none-eabihf
```

Reply:
Good reference for the future, thanks. *bookmarks tweet*